console/console-init/ui/src/modules/address/components/AddressToggleGroup/AddressToggleGroup.tsx

```diff
@@ -127,15 +133,21 @@ const AddressToggleGroup: React.FunctionComponent<IAddressToggleGroupProps> = ({
           )}
         </ToolbarFilter>
       </ToolbarItem>
-      <ToolbarItem spacer={{ md: "spacerNone" }} data-codemods="true">
+      <ToolbarItem
+        spacer={{ md: "spacerNone" }}
+        data-codemods="true"
+        id="addr-toggle-type-toolbar-item"
+      >
         <ToolbarFilter
           chips={typeSelected ? [typeSelected] : []}
           deleteChip={onDelete}
           categoryName="Type"
+          id="addr-toggle-type-toolbar-filter"
         >
           {filterSelected && filterSelected.toLowerCase() === "type" && (
             <DropdownWithToggle
-              id="al-filter-select-type-dropdown"
+              id="addr-toggle-type-dropdown"
```

Comment:
Please revisit this id

Reply:
I have used the following format:
addr-toggle: name of component
type: functonality of selecting type from dropdown
dropdown: the element

Reply:
rename this as 'addr-togglegrp-addresstype-dropdown

Reply:
Will do.